tcms/templates/case/get_details.html

```diff
@@ -1,14 +1,15 @@
+{% load i18n %}
 {% load comments %}
 {% load extra_filters %}
 <td id="id_case_text_{{ forloop.counter }}" colspan="12">
 	<table class="case_detail">
 		<tr>
 			<td valign="top" colspan='2'>
-				<h4>Text:</h4>
+                            <h4>{% trans "Text:" %}</h4>
```

Comment:
Trailing `:` in string. Elsewhere we've excluded these from the source strings.

Reply:
ok